validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendSignedBlockRequest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.remote.typedef.handlers;
+
+import static tech.pegasys.teku.validator.remote.apiclient.ValidatorApiMethod.SEND_SIGNED_BLINDED_BLOCK;
+import static tech.pegasys.teku.validator.remote.apiclient.ValidatorApiMethod.SEND_SIGNED_BLOCK;
+
+import java.util.Collections;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.spec.datastructures.blocks.SignedBeaconBlock;
+import tech.pegasys.teku.validator.api.SendSignedBlockResult;
+import tech.pegasys.teku.validator.remote.apiclient.ValidatorApiMethod;
+import tech.pegasys.teku.validator.remote.typedef.ResponseHandler;
+
+public class SendSignedBlockRequest extends AbstractTypeDefRequest {
+
+  public SendSignedBlockRequest(final HttpUrl baseEndpoint, final OkHttpClient okHttpClient) {
+    super(baseEndpoint, okHttpClient);
+  }
+
+  public SendSignedBlockResult sendSignedBlock(SignedBeaconBlock signedBeaconBlock) {
+    final ValidatorApiMethod apiMethod =
+        signedBeaconBlock.getMessage().getBody().isBlinded()
+            ? SEND_SIGNED_BLINDED_BLOCK
+            : SEND_SIGNED_BLOCK;
+    return postJson(
+            apiMethod,
+            Collections.emptyMap(),
+            signedBeaconBlock,
+            signedBeaconBlock.getSchema().getJsonTypeDefinition(),
+            new ResponseHandler<>())
+        .map(__ -> SendSignedBlockResult.success(Bytes32.ZERO))
```

Comment:
you're not fully handling the response right?

Reply:
by this point it either has an internal block, or it failed... the post doesn't get data back unless there's an error...

Reply:
I think we used to return `Bytes32.ZERO` as the root here anyway, but we could actually provide the real block root from `signedBeaconBlock.hashTreeRoot()` here which would probably be slightly better in case we ever do display that root somewhere.